src/main/java/com/cisco/trex/stateless/TRexTransport.java

```diff
@@ -142,6 +142,21 @@ public synchronized String sendJson(String json) {
     }
     byte[] msg = zmqSocket.recv();
 
+    if (msg == null) {
```

Comment:
to get accurate condition of handling this case, it can use zmqSocket.errno() to check if the failed response is caused by the specified error(lost connection?)
for more utilities, check implementation of ZMQ.java